ci-operator/step-registry/gather/installer/gather-installer.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: installer
+  from: installer
+  commands: installer.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    Gather installation logs in case of failed installation.
```

Comment:
This will always attempt to gather, not just in the case of a failed installation

Reply:
Added:
  optional_on_success: true
  best_effort: true